generateUser.php

```diff
@@ -0,0 +1,23 @@
+<?php
```

Comment:
This file should be removed from the branch

Reply:
removed